tests/zfs-tests/cmd/libzfs_input_check/libzfs_input_check.c

```diff
@@ -37,7 +37,7 @@ const char *active_test;
 /*
  * Tracks which zfs_ioc_t commands were tested
  */
-boolean_t ioc_tested[256];
+boolean_t ioc_tested[ZFS_IOC_LAST - ZFS_IOC_FIRST];
```

Comment:
i'd like to see separate
#define ZFS_IOC_ALL (ZFS_IOC_LAST - ZFS_IOC_FIRST)

Reply:
This entire file uses `<something> - ZFS_IOC_FIRST` for cmd_tested[] index, so this is no less clear than having extra macro just for it.